lib/base-compiler.js

```diff
@@ -55,6 +56,11 @@ Compile.prototype.newTempDir = function () {
 Compile.prototype.writeFile = Promise.denodeify(fs.writeFile);
 Compile.prototype.readFile = Promise.denodeify(fs.readFile);
 Compile.prototype.stat = Promise.denodeify(fs.stat);
+Compile.prototype.couldHaveOptOutput = function(options, version) {
+    //TODO(jared): a compiler meta data gatherer could improve this
+    return version.toLowerCase().indexOf("clang") > -1 && 
+          options.some((x) => x == "-fsave-optimization-record");
```

Comment:
Let's merge as-is and work on improvements in `master`. I'd like it for the compiler drop-down too (so you can filter better by compiler type etc)